File: GO/Iniciante/1017.go
package main

import (
	"fmt"
)

func main() {

  var tempo, vel_media float64
  fmt.Scan(&tempo)
  fmt.Scan(&vel_media)
  
  litros := vel_media / 12.0
  litros *= tempo

  fmt.Printf("%.3f\n", litros)

}

File: C++/Iniciante/1017.cpp
#include <stdio.h>
#include <iostream>

int main() {

  int tempo, vel_media;
  double distancia;
  std::cin >> tempo;
  std::cin >> vel_media;
  
  distancia = (vel_media / 12.0) * tempo;

  printf("%.3lf\n", distancia);
    
}
